src/splitcap.jl

```diff
@@ -46,7 +47,7 @@ function splitcap(
                 output = get!(outputs, dst) do
                     stream = key2stream(dst)
                     buffer = sizehint!(UInt8[], buffer_size + 1500)
-                    append!(buffer, reader.raw_header)
+                    own_streams && append!(buffer, reader.raw_header)
```

Comment:
I'm not sure, but maybe something like will work?
```suggestion
                    position(stream) == 0 && append!(buffer, reader.raw_header)
```